test/test_integration_credentials.py

```diff
@@ -248,58 +256,101 @@ def test_https_logout_twice_returns_could_not_logout_message(self):
                                               ['logout'], exit_code=1)
 
         self.assertIn("Failed to log out. You are already logged out", unsuccessful_logout.strip())
```

Comment:
we should make sure that the keyring is empty as well just as we do for the netrc